### ISSUE TITLE
Cancer Type Summary: How to deal with multiple alterations?

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/cbioportal#3239]
Cancer Type Summary is really useful to get an overall statistics about alterations in different cancer types. But in cases with multiple alterations, is there any way we can render the data with more details?

![image](https://user-images.githubusercontent.com/840895/31357763-1959424e-ad11-11e7-9c67-8134df1ce17a.png)

![image](https://user-images.githubusercontent.com/840895/31357854-6e260b5e-ad11-11e7-99b6-e8a849542792.png)

**Comment by jjgao** [comment]
I could not resist it and created a mockup to replace the current bars with oncoprint :) 

![image](https://user-images.githubusercontent.com/840895/31358266-e217cfba-ad12-11e7-94d0-28df53331078.png)

I am thinking of the possibility that we move this plot into the OncoPrint tab. By default, it breaks the oncoprint by cancer type, but one can choose to break it by any other clinical attributes, e.g. tumor stage or gender.

**Comment by schultzn** [comment]
I understand the temptation. But maybe too much for this release? And probably too complicated for most users? This would have to be an advanced version of the graph.

**Comment by ecerami** [comment]
This is future idea, right? e.g. not for this release.

**Comment by jjgao** [comment]
This is for future.

**Comment by tmazor** [comment]
I like the idea of breaking down the multiple alterations, but I do also worry that it would get too complicated. One of the nice things about the cancer summary tab is that it is a very simple overview - perhaps there's an intermediate color scheme between an oncoprint-esque bar and the current grey bar.

I think the option to break out the bars by other clinical attributes could be a very useful feature -- perhaps this becomes the more advanced option, with full oncoprint coloring and the ability to change the x-axis groupings.

**Comment by jjgao** [comment]
@schultzn @tmazor Good point on the complexity. Even if the user can understand it (together with oncoprint), when they put it in slides, it's not easy to digest. There should be options to control the details of the chart (maybe the same options for oncoprint, esp if on the same tab?).

**Comment by alisman** [comment]
@jjgao can we close this?

**Comment by jjgao** [comment]
@alisman let's keep it in the icebox for now.
